tests/Benchmarks/System.Text.Primitives/PrimitiveParserSBytePerfTests.cs

```diff
@@ -13,58 +13,6 @@ namespace System.Text.Primitives.Benchmarks
 {
     public partial class PrimitiveParserPerfTests
     {
-        private static readonly string[] s_SByteTextArray = new string[17]
-        {
-           "95",
-            "2",
-            "112",
-            "-112",
-            "-21",
-            "-2",
-            "114",
-            "-114",
-            "-124",
-            "117",
-            "-117",
-            "-14",
-            "14",
-            "74",
-            "21",
-            "83",
-            "-127"
-        };
-        
-        public IEnumerable<object> SByte_BytesConsumed_VariableLength_Arguments()
-            => s_Int32TextArray.Select(text => new Utf8ByteArrayArgument(text));
-
-        [Benchmark]
-        [ArgumentsSource(nameof(SByte_BytesConsumed_VariableLength_Arguments))]
-        public bool PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength(Utf8ByteArrayArgument text)
-            => Utf8Parser.TryParse(text.CreateSpan(), out sbyte value, out int bytesConsumed);
-        
-        [Benchmark]
-        [ArgumentsSource(nameof(SByte_BytesConsumed_VariableLength_Arguments))]
-        public bool PrimitiveParserByteSpanToSByte_BytesConsumed_VariableLength_Baseline(Utf8ByteArrayArgument text)
-            => sbyte.TryParse(text.Text, out sbyte value);
-        
-        public IEnumerable<object> PrimitiveParserByteSpanToSByte_Arguments()
-        {
-            yield return new Utf8ByteArrayArgument("107"); // standard parse
-            yield return new Utf8ByteArrayArgument("127"); // max value
-            yield return new Utf8ByteArrayArgument("0"); 
-            yield return new Utf8ByteArrayArgument("-128"); // min value
-            yield return new Utf8ByteArrayArgument("-000000000000000000001235abcdfg");
-            yield return new Utf8ByteArrayArgument("-2147abcdefghijklmnop");
-        }
-
-        [Benchmark]
-        [ArgumentsSource(nameof(PrimitiveParserByteSpanToSByte_Arguments))]
```

Comment:
Given we are keeping the Thai, CustomParser tests, I would keep this particular test and mark the benchmark as baseline. This gives us a good measurement to compare the custom parser against.

`[Benchmark(Baseline=true)]`

Reply:
Also, let's keep the test cases identical:
๑๑๑ => 111
๑๒๔ => 124
๒ => 2
๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๐๑๒๓๕abcdfg =>000000000000000000001235abcdefg
๒๑๔๗๔abcdefghijklmnop =>21474abcdefghijklmnop

https://en.wikipedia.org/wiki/Thai_numerals#Main_numbers